src/AspNet.Security.OAuth.Validation/Events/MessageReceivedContext.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using Microsoft.AspNetCore.Authentication;
 using Microsoft.AspNetCore.Http;
 
-namespace AspNet.Security.OAuth.Validation.Events
+namespace AspNet.Security.OAuth.Validation
 {
-    public class ReceivingTokenContext : BaseControlContext
+    public class MessageReceivedContext : BaseControlContext
```

Comment:
Feel free to differ from the events names used for the JWT bearer middleware if you have better names in mind. The Azure AD guys pushed hard to keep the events names similar to the OIDC client middleware but I don't think it's the right thing to do.